Tensile/Source/client/source/Reference.cpp

```diff
@@ -82,29 +81,29 @@ namespace Tensile
 
                 bool rv = anchorRelCoord < zp.padStart || anchorRelCoord >= elementEdge;
 
-                bool checkUnsignedRangeOpt = false;
```

Comment:
cppcheck: Condition 'checkUnsignedRangeOpt' is always false
I only commented this out because maybe it's useful for debugging

Reply:
Looking closer, this should probably be turned into a unit test or removed.

Reply:
I think you need to make the variable `const` for cppcheck not to warn.

Reply:
I believe the root of the problem is that the code is unused. I'm not sure how using `const` would fix that. I've just updated the PR to remove this code.